src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.event;
+
+import org.candlepin.subscriptions.db.EventRecordRepository;
+import org.candlepin.subscriptions.db.model.EventRecord;
+import org.candlepin.subscriptions.json.Event;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.springframework.stereotype.Controller;
+
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+import javax.persistence.EntityNotFoundException;
+import javax.transaction.Transactional;
+
+/**
+ * Encapsulates interaction with event store.
+ */
+@Controller
```

Comment:
I think the `@Service` annotation might be more appropriate here.  Traditionally `@Controller` annotation is used to indicate a class is a web controller.

Reply:
@lindseyburnett Good catch! Will change in a follow-up commit.

Reply:
Done